internal/agent/device/systeminfo/system_info.go

```diff
@@ -29,8 +29,8 @@ const (
 	ipv4RoutePath    = "/proc/net/route"
 	ipv6RoutePath    = "/proc/net/ipv6_route"
 
-	// ScriptOverrideDir is the directory where custom/overide scripts are stored
-	ScriptOverrideDir = "collect.d"
+	// DetailsDir is the directory where custom/overide scripts are stored in the agent data dir
+	DetailsDir = "details.d"
```

Comment:
note: I don't love this but I also don't think its appropriate to include these in $PATH

Reply:
I think a `plugins` dir would be more intuitive so `plugins/system-info`